packages/components/src/components/ViewYAML/ViewYAML.js

```diff
@@ -15,20 +15,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import jsYaml from 'js-yaml';
 import classNames from 'classnames';
+import { Prism as SyntaxHighlight } from 'react-syntax-highlighter';
 
 const ViewYAML = props => {
-  const { className, dark, resource } = props;
+  const { className, dark, resource, enableSyntaxHighlighting = false } = props;
 
   return (
-    <div
+    <SyntaxHighlight
       className={classNames('bx--snippet--multi', className, {
-        'tkn--view-yaml--dark': dark
+        'tkn--view-yaml--dark': dark,
+        'tkn--view-yaml--enable-syntax-highlighting': enableSyntaxHighlighting
       })}
+      codeTagProps={{}}
+      language="yaml"
+      useInlineStyles={false}
     >
-      <code>
-        <pre>{jsYaml.dump(resource)}</pre>
-      </code>
-    </div>
+      {jsYaml.dump(resource).trim()}
```

Comment:
Was the `trim()` added for a particular reason? It would be good to understand if this is necessary.

Reply:
I found that it has some extra bottom padding compare to the others. It fixes after I trim it. oTZ

Reply:
It's probably a trailing newline character which we should leave if possible, so we're accurately rendering the content provided.

Reply:
Ok. So I will remove the trim out of it.

Reply:
@AlanGreene I think it's not because of final new line in yaml string but it's because of syntax highlight itself. Try this example from storybook:

Raw yaml code:

<img width="811" alt="Screen Shot 2564-06-09 at 23 02 47" src="https://user-images.githubusercontent.com/484530/121389617-d01fc900-c976-11eb-87cd-645740e731a9.png">

Syntax highlight enable:

<img width="809" alt="Screen Shot 2564-06-09 at 23 03 12" src="https://user-images.githubusercontent.com/484530/121389684-de6de500-c976-11eb-9bb9-e92805274890.png">

And if we try react syntax highlighter demo site https://react-syntax-highlighter.github.io/react-syntax-highlighter/demo/. We'll see the same (note that they have final new line in editor, we must remove it first and see it).